recipes/seqan_tcoffee/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version="1.13.7" %}
+
+build:
+  number: 0
+
+about:
+  home: https://github.com/seqan/seqan/blob/master/apps/seqan_tcoffee/README
+  license: https://github.com/seqan/seqan/blob/master/apps/seqan_tcoffee/LICENSE
+  summary: SeqAn::T-Coffee - Multiple Sequence Alignment.
+
+package:
+  name: seqan_tcoffee
+  version: {{ version }}
+
+source:
+  url: http://packages.seqan.de/seqan_tcoffee/seqan_tcoffee-{{ version }}-Mac-x86_64.zip # [osx]
```

Comment:
Can this be compiled, instead of downloaded?

Reply:
Yes, it can be also compiled. 
Do you prefer the compiled version? 
If so I can update the yaml and run a new pull request

Reply:
Yes, the source version is preferred. You can just use this branch and commit the updated yaml and build.sh script.